script/app.js

```diff
@@ -56,6 +71,14 @@ const renderTodos = () => {
     reversedTodos.forEach((task) => {
         const taskCard = getTodoCard(task);
         taskListContainerElement.appendChild(taskCard);
+        if (task.isCompleted) {
+            const completeButton = taskCard.querySelector(
+                ".task-card__icon--complete"
+            );
+            completeButton.style.display = "none";
+            const editButton = taskCard.querySelector(".task-card__icon--edit");
+            editButton.style.display = "none";
```

Comment:
aren't we  have to disable the button  or we have to make it dissapear?

Reply:
Following the mock-up, it should disappear.